docker-compose.yml

```diff
@@ -1,18 +1,16 @@
-version: "3"
-
 services:
   libretranslate:
     container_name: libretranslate
     build:
       context: .
-      dockerfile: docker/Dockerfile
+      dockerfile: ./docker/Dockerfile
```

Comment:
This was for removing a problem related to the path of `/app`, that I'm no longer able to replicate. Maybe it was just a problem in my configuration. It still works this way, but maybe it's not necessary.